servicex/transformer/servicex_adapter.py

```diff
@@ -54,5 +77,17 @@ def put_file_complete(self, file_path, file_id, status,
             "avg-rate": avg_rate
         }
         print("------< ", doc)
+
         if self.server_endpoint:
-            requests.put(self.server_endpoint + "/file-complete", json=doc)
+            success = False
+            attempts = 0
+            while not success and attempts < 3:
```

Comment:
Same question here - shouldn't this just use Retry rather than implementing our own retry mechanism?